UT4MasterServer/Controllers/AccountController.cs

```diff
@@ -218,17 +218,17 @@ public IActionResult GetSSODomains()
 	[AllowAnonymous]
 	public async Task<IActionResult> RegisterAccount([FromForm] string username, [FromForm] string email, [FromForm] string password, [FromForm] string recaptchaToken)
 	{
-		var reCaptchaSecret = this.configuration.GetValue<String>("ReCaptcha:SecretKey");
+		var reCaptchaSecret = configuration.GetValue<string>("ReCaptcha:SecretKey");

```

Comment:
Make ReCaptchaSettings class, bind it to ReCaptcha section in appsettings, and use IOptions to inject it.